src/dependency-manager/src/schema/spec/transform/resource-transform.ts

```diff
@@ -48,7 +48,12 @@ export const transformResourceSpecEnvironment = (environment: Dictionary<Environ
     if (v === undefined || v === null) {
       continue;
     }
-    output[k] = `${v}`;
+
+    if (v instanceof Object) {
```

Comment:
Also I don't think this is sufficient to check if it's an array. And I'm assuming we'll also want to stringify arrays, correct?

Reply:
it is